packages/manager/modules/storages/src/netapp/netapp.template.html

```diff
@@ -0,0 +1,123 @@
+<div class="" style="overflow: auto;">
+    <oui-header
+        data-heading="{{:: 'netapp_product_name' | translate }} - {{ $ctrl.netappId }}"
+    >
+        <oui-header-tabs>
+            <oui-header-tabs-item
+                data-href="{{ $ctrl.serviceLink }}"
+                data-active="$ctrl.currentActiveLink() === $ctrl.serviceLink"
+                ><span data-translate="netapp_tab_global_news"></span>
+            </oui-header-tabs-item>
+            <oui-header-tabs-item
+                data-href="{{ $ctrl.shareLink }}"
+                data-active="$ctrl.currentActiveLink() === $ctrl.shareLink"
+                ><span data-translate="netapp_tab_share"></span>
+            </oui-header-tabs-item>
+        </oui-header-tabs>
+        <oui-guide-menu data-text="{{:: 'netapp_guides_header' | translate }}">
+            <oui-guide-menu-item href="{{ $ctrl.guideUrl }}" data-external>
+                <span data-translate="netapp_guides_header_all_guides"></span>
+            </oui-guide-menu-item>
+        </oui-guide-menu>
+    </oui-header>
+
+    <div class="px-3 px-md-4" data-ui-view>
```

Comment:
```suggestion
    <div class="container-fluid" data-ui-view>
```
As some `class="row"` is used below